core/cap-0051.md

```diff
@@ -153,6 +165,10 @@ func $call2(param $contract i64) (param $func i64) (param $arg1 i64) (param $arg
 func $call3(param $contract i64) (param $func i64) (param $arg1 i64) (param $arg2 i64) (param $char i64) (result i64)
 
 func $call4(param $contract i64) (param $func i64) (param $arg1 i64) (param $arg2 i64) (param $arg3 i64) (param $arg4 i64) (result i64)
+
+/// Calls another function $func in a contract $contract with variadic arguments stored in a vector $args_vec.
+/// If the call is successful, it forwards the result of the called function. Otherwise, it traps.
+func $call(param $contract i64) (param $func i64) (param $args_vec i64) (result i64)
```

Comment:
Do we need call1 to 4 if we have this?

Reply:
Probably not, but I'll do some experimentation before confirming and removing them.